packages/console/src/pages/Security/CreateCaptchaForm/constants.ts

```diff
@@ -5,14 +5,46 @@ import turnstile from '@/assets/images/turnstile.svg?react';
 
 import { type CaptchaProviderMetadata } from './types';
 
+const reCAPTCHAEnterpriseReadme = `
+# reCAPTCHA Enterprise
+
+reCAPTCHA Enterprise is a Google service that protects websites from fraud and abuse using advanced bot detection without disrupting user experience. This guide will walk you through the process of setting up reCAPTCHA Enterprise with Logto.
+
+## Prerequisites
+
+- A Google Cloud project
+
+## Setup a reCAPTCHA key
+
+1. Go to the [reCAPTCHA page of Google Cloud Console](https://console.cloud.google.com/security/recaptcha).
+2. Click **Create key** button near "reCAPTCHA keys".
+3. Fill out the form with the following details:
+   - **Display name**: Any name you want to give to the key
+   - **Application type**: Website
+   - **Domain list**: Add Logto's endpoint domain
+4. After creating the key, you will be redirected to the key details page, copy the **ID**.
+
+## Setup an API key
+
+1. Go to the [Credentials page of Google Cloud Console](https://console.cloud.google.com/apis/credentials).
+2. Click **Create credentials** button and select **API key**.
+3. Copy the API key.
+4. Optionally, you can restrict the API key to \`reCAPTCHA Enterprise API\` to make it more secure.
+5. Remember to leave "Application restrictions" to **None** if you don't understand what it is.
```

Comment:
I think we can use the same style for these two
![image](https://github.com/user-attachments/assets/391dcc4c-a21c-4df1-9b7e-76060472ed8c)